README.md

```diff
@@ -287,20 +299,125 @@ models
     └── config.pbtxt
 ```
 
-## Using Your Own Python Interpreter
+## Using Custom Python Execution Environments
 
 Python backend shipped in the [NVIDIA GPU Cloud](https://ngc.nvidia.com/)
 containers uses Python 3.8. Changing your environment using
```

Comment:
"uses Python 3,8". If your Python model is compatible with Python 3.8 and requires only modules already included in the Triton container, then you can skip this section. If you need to use a different version of Python or if you have additional dependencies, you need to recompile the stub executable and create an execution environment as described below and include that with your model."

I would delete "Changing your..." to the end of the section.